docs/OPENID4VCI-SATOSA-FRONTEND.md

```diff
@@ -285,44 +285,46 @@ Metadata for OpenID Federation.
 | config.credential_configurations.credential_specification | A dictionary where each key corresponds to a credential type declared in `config.metadata.openid_credential_issuer.credential_configurations_supported` (metadata). Each value includes the `expiry_days` (validity in days) and a `template` for rendering the credential with placeholders in given format present in `config.metadata.openid_credential_issuer.credential_configurations_supported.<cred>.format`.  Mandatory in `credential` endpoint. |
 
 ### Security
+
 Root section for security configuration.
 
-| Key                                         | Description                                                                                                       |
-|---------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
-| config.security.dpop_required               | This is the flag to enable the DPOP interoperability and validation, if missing it will be set to true by default |
-| config.security.wallet_attestation_required | This is the flag to enable the wallet_attestation validation, if missing it will be set to true by default        |
+| Key                                         | Description                                                                                                                                                                                                                                                                                                                                                               |
+| ------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| config.security.dpop_required               | This is the flag to enable the DPOP validation needed as access token for token and credential endpoints<br />([https://datatracker.ietf.org/doc/html/rfc9449](https://datatracker.ietf.org/doc/html/rfc9449)).<br /> If this setting is missing it will be set to true by default.                                                                                       |
+| config.security.wallet_attestation_required | This is the flag to enable the wallet client attestation validation in POP JWT format<br />([https://www.ietf.org/archive/id/draft-ietf-oauth-attestation-based-client-auth-06.html#section-5.2](https://www.ietf.org/archive/id/draft-ietf-oauth-attestation-based-client-auth-06.html#section-5.2)).<br />If this setting is missing it will be set to true by default. |
```

Comment:
```suggestion
| config.security.wallet_attestation_required | This flag enables the [OAuth 2.0 Attestation-Based Client Authentication](https://www.ietf.org/archive/id/draft-ietf-oauth-attestation-based-client-auth-06.html). If this setting is missing it will be set to true by default. |
```